### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by addod19**

### CODE REVIEW

**Comment by offorma** [comment]
Well done guys  :+1: , I see you have done a lot of work using scss that is very good. I have only notice that you forgot to validate your Html. Please use this link   https://validator.w3.org/#validate_by_upload  to make sure your Html is valid.  Secondly, some of the Scss files are empty if there are not doing anything please kindly remove them. Also, Notice that your images are a bit stretched on  fullscreen  so you can solve this by making the image responsive eg 
```
.img {    height: auto;
    width: 100%;
}
```
That's all. Please make the changes and submit for review. If you have anything you will like to clarify
I am on slack @Nonso

**Comment by addod19** [comment]
Thank you @nonso...we will work on it and submit a code review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,388 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>YouTube</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
```
Comment (samgaco) [inline index.html:6]: Add a character encoding tag such as ` <meta charset="UTF-8"> `

**Review by samgaco — APPROVED** [review]
#### Hi @addod19 and @rsundar :hand: 

#### You have worked through the suggested changes hence you are approved :+1:  Congratulations :clap: 



------

**Please, before merging don't forget to correct this:**

* You still have one error if submitting the index file to the validator => Error: The character encoding was not declared. Proceeding using windows-1252.


------

### That's it. Congratulations on finishing the last project of this module :100: 

##### You may merge to the master branch and submit the project completion form.

**Comment by addod19** [comment]
Thank you

**Pull request merged**